src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -2266,10 +2393,27 @@ public App ()
             <td>
                 Offline storage is enabled by default. To disable it, add the following feature flag:
 
-                ```typescript
+                ```csharp
                 offlineStorageEnabled: false
                 ```
             </td>
+        </tr>
+               <tr>
+            <td>
+                Enable or disable background reporting when application goes to the background state.
+
+            </td>
+            <td>
+
+                Background reporting is disabled by default. To enable it, add the following feature flag:
+
+                ```csharp
+                 backgroundReportingEnabled: true
+                ```
+                Prerequisites in your iOS app project to use background instrumentation:
+
+                - Add the key `Permitted background task scheduler identifiers` with an array including an item containing your apps bundle Id.
```

Comment:
Need to add an example

Reply:
![Screenshot 2024-06-07 at 11 45 34 AM (1)](https://github.com/newrelic/docs-website/assets/89222514/f4952ff3-a729-4537-87ad-aacc11153c92)

Reply:
```suggestion
                - In your app's `Info.plist` file, add the key "Permitted background task scheduler identifiers". This key should be an array containing a single string value, which is your app's bundle identifier.
```